tests/memory/host/host.cpp

```diff
@@ -143,6 +153,12 @@ int main(int argc, const char* argv[])
     printf("===Starting malloc boundary test.\n");
```

Comment:
```suggestion
    int seed = -1;
    if (argc > 1) {
        // Try to parse seed passed from command-line
        int len = strlen(argv[2]);
        char* endptr = NULL;
        
        seed = strtol(argv[2], endptr, 10);
        if (endptr - argv[2] != len) {
            fprintf(stderr, "%s is not a valid seed\n", argv[2]);
            return 1;
        }
    }

    printf("===Starting malloc boundary test.\n");
```

Reply:
Note you'll have to include stdlib.h and string.h in this file now.

Reply:
looks good, thanks